app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalListItem.vue

```diff
@@ -0,0 +1,415 @@
+                </td>
+                <td>
+                  <woot-button
+                    v-tooltip.top-end="
+                      $t(
+                        'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.AVAILABLE_LOCALES.TABLE.SWAP'
+                      )
+                    "
+                    size="tiny"
+                    variant="smooth"
+                    icon="arrow-swap"
+                    color-scheme="primary"
+                    :disabled="locale.code === selectedLocaleCode"
+                    @click="swapLocale"
+                  />
+                  <woot-button
+                    v-tooltip.top-end="
+                      $t(
+                        'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.AVAILABLE_LOCALES.TABLE.DELETE'
+                      )
+                    "
+                    size="tiny"
+                    variant="smooth"
+                    icon="delete"
+                    color-scheme="secondary"
+                    @click="deleteLocale"
+                  />
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import thumbnail from 'dashboard/components/widgets/Thumbnail';
+import Label from 'dashboard/components/ui/Label';
+export default {
+  components: {
+    thumbnail,
+    Label,
+  },
+  props: {
+    portals: {
+      type: Array,
+      default: () => [],
+    },
+    status: {
+      type: String,
+      default: '',
+      values: ['archived', 'draft', 'published'],
+    },
+    selectedLocaleCode: {
+      type: String,
+      default: '',
+    },
+  },
+  computed: {
+    labelColor() {
+      switch (this.status) {
+        case 'archived':
+          return 'secondary';
+        case 'draft':
+          return 'warning';
+        default:
+          return 'success';
+      }
+    },
+    defaultLocale(code) {
+      return code === this.selectedLocaleCode;
+    },
+  },
+  methods: {
+    addLocale() {
+      this.$emit('add');
+    },
+    openSite() {
+      this.$emit('open-site');
+    },
+    openSettings() {
+      this.$emit('open');
+    },
+    swapLocale() {
+      this.$emit('swap');
+    },
+    deleteLocale() {
+      this.$emit('delete');
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.portal {
+  background-color: var(--white);
+  padding: var(--space-normal);
+  border: 1px solid var(--color-border-dark);
+  border-radius: var(--border-radius-medium);
+  position: relative;
+  display: flex;
+  margin-bottom: var(--space-normal);
+
+  .container {
+    margin-left: var(--space-one);
+    flex-grow: 1;
+
+    header {
+      display: flex;
+      align-items: flex-start;
+      justify-content: space-between;
+      margin-bottom: var(--space-normal);
+      .title-status--wrap {
+        display: flex;
+        align-items: center;
+        .status {
+          margin: 0 0 0 var(--space-one);
```

Comment:
Use margin left and `space-small`

Reply:
@nithindavid  The label component has some margins on the bottom and right that's why I added it. 

<img width="478" alt="image" src="https://user-images.githubusercontent.com/64252451/182313819-8c6c21de-e895-461c-96f6-ac651b798b44.png">